ee/src/main/java/org/jboss/as/ee/subsystem/ManagedThreadFactoryResourceDefinition.java

```diff
@@ -52,9 +60,10 @@
 
     public static final SimpleAttributeDefinition CONTEXT_SERVICE_AD =
             new SimpleAttributeDefinitionBuilder(CONTEXT_SERVICE, ModelType.STRING, true)
-                    .setAllowExpression(false)
+                    .setAllowExpression(true)
```

Comment:
This change requires a new model version and transformers to reject models/operations containing expressions for older versions.

Reply:
this change was required by current tests, which has expressions, think that probably then makes sense to not use expressions on tests

Reply:
reverted, fixing subsystem tests instead